report/3.Theory/SyntaxTheory.tex

```diff
@@ -0,0 +1,178 @@
+% This document needs to include:
```

Comment:
Generelt til denne: Ensure that all abbreviations are present in the Abbreviations.tex file

Reply:
All should be added to abbreviations now